exercises/Python_Principles_4.ipynb

```diff
@@ -1097,7 +986,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this example, we can access both the index and list items separately using enumerate using index notation on the tuples that enumerate returns."
+    "In this example, we can acces both the index and list items separately using enumerate using index notation on the tuples that enumerate returns."
```

Comment:
@AlexTeslenko is this a typo acces instead of access? What do you think?

Reply:
yep, I'm not sure why s was deleted from your original text, by accident most probably.